src-tauri/src/command.rs

```diff
@@ -17,10 +19,19 @@ pub struct CommandMessage {
 pub async fn execute_command(
     CommandMessage { command, payload }: CommandMessage,
     _app_state: tauri::State<'_, AppState>,
+    window: tauri::Window,
 ) -> Result<TransferState, String> {
-    info!("command `{}` received with payload `{}`", command, payload);
+    info!("received command `{}` with payload `{}`", command, payload);
 
     match command.as_str() {
+        "[INIT] Get initial state" => {
+            let initial_state = AppState::new(StateStatus::Stable);
+            let transfer_state = TransferState::from(initial_state);
+            window.emit("state-changed", transfer_state).unwrap();
+            // TODO: should not be necessary to create new, allow Copy trait on TransferState?
+            info!("emitted event `{}` with payload `{:?}`", "state-changed", TransferState::from(AppState::new(StateStatus::Stable)));
+            Ok(TransferState::from(AppState::new(StateStatus::Stable)))
```

Comment:
I think passing the value by reference instead of copying the state is sufficient in this case, e.g.:
```suggestion
            let transfer_state = TransferState::from(AppState::new(StateStatus::Stable));
            info!(
                "emitted event `{}` with payload `{:?}`",
                "state-changed", &transfer_state
            );
            Ok(transfer_state)
```